tests/installation/skip_registration.pm

```diff
@@ -49,6 +49,7 @@ sub run {
         );
     }
     skip_registration;
+    wait_still_screen;
```

Comment:
This waiting time of at least 7 seconds is a big waste of time. I suggest to not have this code stand as is, not even with a comment explaining the investment of time. Can we not report this as a bug – because I have also seen that as well as other people but I think you understood it better now – and try to detect it gracefully – apparently you managed to "detect" it :) – and record a soft failure and continue unless we really want it to stay? If we fail to match it in time we should handle that gracefully and just go on.

Reply:
Understood. Filed https://bugzilla.suse.com/show_bug.cgi?id=1102705. In the meantime I'll try to figure out more "clever" approach.